.solhint.json

```diff
@@ -2,7 +2,7 @@
   "extends": "solhint:recommended",
   "plugins": [],
   "rules": {
-    "compiler-version": ["warn", "^0.8.19"],
+    "compiler-version": ["warn", "^0.8.15"],
```

Comment:
since we use "^", it's safer to use the lower version to avoid meaningless warnings